terraform/gcp/main.tf

```diff
@@ -355,9 +355,10 @@ module "flink" {
   env                            = var.env
   building_block                 = var.building_block
   flink_container_registry       = var.flink_container_registry
-  flink_release_name             = var.flink_release_name
-  flink_image_name               = var.flink_image_name
   flink_image_tag                = var.flink_image_tag
+  flink_merged_pipeline_release_names = var.flink_merged_pipeline_release_names
```

Comment:
The list of variables need to be formatted properly.

Reply:
Formatted accordingly